Sources/DangerSwiftLint/DangerSwiftLint.swift

```diff
@@ -8,10 +8,10 @@ public struct SwiftLint {
     /// This is the main entry point for linting Swift in PRs using Danger-Swift.
     /// Call this function anywhere from within your Dangerfile.swift.
     @discardableResult
-    public static func lint() -> [Violation] {
+    public static func lint(directory: String? = nil, configFile: String? = nil) -> [Violation] {
```

Comment:
I'm not sure the `configFile` name is so pretty, I don't super-like it but I don't have a nicer idea at the moment 🤔

Reply:
I'm all ears for a better name 😄 didn't think of a better one as well

Reply:
Not sure if a good alternative exists. It's the parameter used to specify the file where the config is 🙃